### ISSUE TITLE
code-server:php

### ISSUE THREAD

**Issue by toxic0berliner** [linuxserver/docker-mods#493]
Using the code-server mod-php is failing at container start and preventing code-server to run.

The only offending lines I find in the logs are : 
```
/etc/s6-overlay/s6-rc.d/init-mod-code-server-php-add-package/run: line 10: syntax error: unexpected end of file
s6-rc: warning: unable to start service init-mod-code-server-php-add-package: command exited 2
```

My compose file : 

```
version: "3"
services:
  codeserver:
    container_name: codeserver
    image: linuxserver/code-server:latest
    hostname: codeserver
    labels:
      traefik.http.services.codeserver.loadbalancer.server.port: 8443
      traefik.http.services.codeserver.loadBalancer.server.scheme: http
      traefik.http.routers.codeserver.rule: Host(`ide.example.com`)
    volumes:
      - codeserver-config:/config
      - /var/run/docker.sock:/var/run/docker.sock
    environment:
      - PUID=1234
      - PGID=123
      - UMASK=002
      - TZ=Europe/Paris
      - SUDO_PASSWORD=${SUDO_PASSWORD} #optional
      - PROXY_DOMAIN=ide.example.com #optional
      - DEFAULT_WORKSPACE=/config/workspace #optional
      - DOCKER_MODS=linuxserver/mods:universal-docker|linuxserver/mods:code-server-extension-arguments|linuxserver/mods:code-server-shellcheck|linuxserver/mods:code-server-nodejs|linuxserver/mods:code-server-npmglobal|linuxserver/mods:code-server-python3|linuxserver/mods:code-server-dotnet|linuxserver/mods:code-server-golang|linuxserver/mods:code-server-java11|linuxserver/mods:code-server-php|linuxserver/mods:universal-package-install
      - VSCODE_EXTENSION_IDS=vscode-icons-team.vscode-icons|ms-azuretools.vscode-docker|donjayamanne.git-extension-pack|mhutchie.git-graph|redhat.ansible
      - INSTALL_PACKAGES=wget|vim|zip|apt-transport-https|ansible|dnsutils|iputils-ping|netcat
      - GITLAB_WORKFLOW_INSTANCE_URL=${GITLAB_INSTANCE}
      - GITLAB_WORKFLOW_TOKEN=${GITLAB_TOKEN}
      - VISUAL=vim
      - EDITOR="$VISUAL"
    restart: unless-stopped
    networks:
      traefiknet:
        aliases: 
          - codeserver
volumes:
  codeserver-config:
networks:
  traefiknet:
    name: traefiknet
    driver: bridge
    external: true
```

The exact same compose file only removing the `linuxserver/mods:code-server-php` works just fine, replacing it with `linuxserver/mods:code-server-php8` also works fine...

**Comment by aptalca** [comment]
fix in #494

**Comment by toxic0berliner** [comment]
wow, impressive reactivity ! Not sure if I can test it right away, will finish my coding session first as container takes a while to restart but will test asap and confirm ;)

Thanks a lot !
